Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py

```diff
@@ -147,15 +145,20 @@ def tags_arg_to_request_format(tags):
         } for tag in tags]
     except IndexError:
         return_error(bad_arg_msg)
+    return None
 
 
 ''' INTEGRATION COMMANDS '''
 
 
-def test_connection(client, params):
-    if not client.ms_client.managed_identities_client_id \
-        and (params.get('self_deployed', False) and not params.get('client_credentials')
-             and not (params.get('credentials_auth_code', {}).get('password') or params.get('auth_code'))):
+def test_connection(client: Client, params: dict[str, Any]) -> str:
+    if (
+        not client.ms_client.managed_identities_client_id
+        and params.get('self_deployed', False)
+        and not params.get('client_credentials')
+        and not params.get('credentials_auth_code', {}).get('password')
+        and not params.get('auth_code')
+    ):
         return_error('You must enter an authorization code in a self-deployed configuration.')
```

Comment:
why not raise? The best practice for return_error is using it once, in the main function.

Reply:
same,  I didn't write it... 🫣
do you think I should change it?

Reply:
Yes, it is old standard so you can change it- at least in the commands you are touching anyway.

Reply:
Done, changed in all command to `raise DemistoException`